kubernetes/README.md

```diff
@@ -62,7 +62,7 @@ torchserve:
 deployment:
   replicas: 1 # Changes this to number of node in Node Group
 
-persitant_volume:
+persistant_volume:
```

Comment:
well this is also a typo XD - I believe it should be `persistent_volume` - also can you confirm that you've tested that this change works?